### PR TITLE
Add sample usage of elisctl library within Python code

### PR DESCRIPTION
**Pull request by evelyn9191**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,17 +30,58 @@ Install the package from PyPI:
...
+
+def export_documents():
+    with APIClient(context=None, user=username, password=password) as rossum:
+        response = rossum.get(f"queues/{queue_id}/export?"
+                               f"status={reviewed_documents}"
+                               f"&format={export_format}"
+                               f"&exported_at_after={date_start.isoformat()}"
+                               f"&exported_at_before={date_end.isoformat()}"
+                               f"&ordering=exported_at"
+                               f"&page_size=100&page=1")
```
Comment (pasky) [inline README.md]: Shouldn't we take this chance and use get_paginated() (or whatever it's called)? That is where the huge advantage of using the elisctl library actually comes in. We can then link this from our Python export example at dev hub, what do you think?

File: README.md [hunk 2]
```diff
@@ -30,17 +30,58 @@ Install the package from PyPI:
...
+import json
+
+from elisctl.lib.api_client import APIClient
+from datetime import date, timedelta
+
+queue_id = 12673
+username = 'your_username'
+password = 'your_password'
+reviewed_documents = "exported,exporting,failed_export"
+export_format = "json"
```
Comment (pasky) [inline README.md]: Would the code even work with another export_format, given how the data is saved?

File: README.md [hunk 3]
```diff
@@ -30,17 +30,58 @@ Install the package from PyPI:
...
+
```
Comment (pasky) [inline README.md:56]:
```suggestion

# This example downloads data for documents exported during the previous calendar day.
```

**Review by pasky — CHANGES_REQUESTED** [review]
Great idea. What do you think about my comments?

File: README.md (hunk 2 above)
Comment (evelyn9191) [reply README.md]: Good catch, thanks!

**Review by pasky — APPROVED** [review]
Awesome

**Pull request merged**